bundles/org.openhab.core.thing/src/main/resources/OH-INF/i18n/SystemProfiles.properties

```diff
@@ -23,4 +23,5 @@ profile.config.system.timestamp-offset.offset.label = Offset
 profile.config.system.timestamp-offset.offset.description = Offset to be applied on the state towards the item. The negative offset will be applied in the reverse direction.
 profile.config.system.timestamp-offset.timezone.label = Time Zone
 profile.config.system.timestamp-offset.timezone.description = A time zone to be applied on the state.
+profile-type.system.timestamp-trigger.label = Timestamp on Trigger
```

Comment:
```suggestion
profile-type.system.timestamp-trigger.label = Timestamp on trigger
```

Reply:
We changed spelling for other timestamp Profiles to camel case in https://github.com/openhab/openhab-core/pull/2353 (e.g. "Timestamp on Change"). I would like to keep it.